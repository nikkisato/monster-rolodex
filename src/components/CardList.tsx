import * as React from "react"
import Card from "./Card";
import { IMonster } from "../containers/App"

const CardList = ({ monsters }: { monsters: Array<IMonster>}) => {
	return (
		<div>
			{
				monsters.map((user, i) => {
					return(<Card
						key={i}
						id={monsters[i].id}
						name={monsters[i].name}
						email={monsters[i].name}
					/>
					)
				})
			}
		</div>
	)
}

export default CardList;