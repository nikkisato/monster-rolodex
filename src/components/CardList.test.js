import { shallow } from 'enzyme';
import React from 'react';
import CardList from './CardList';

it('expects to render cardList Component', () => {
  const mockMonsters = [
    {
      id: 1,
      name: 'john snow',
      username: 'JohnJohn',
      email: 'john@gmail.com',
    },
  ];
  expect(
    shallow(<CardList monsters={mockMonsters} />).length
  ).toMatchSnapshot();
});
