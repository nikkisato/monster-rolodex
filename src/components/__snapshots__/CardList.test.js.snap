// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expects to render cardList Component 1`] = `1`;
